src/statsmodels/formula.jl

```diff
@@ -48,8 +48,10 @@ type ModelFrame
     contrasts::Dict{Symbol, ContrastsMatrix}
 end
 
-type ModelMatrix{T <: @compat(Union{Float32, Float64})}
-    m::Matrix{T}
+typealias ModelMatrixContainer{T<:AbstractFloat} AbstractMatrix{T}
```

Comment:
I think it would be clearer if it was called `AbstractFloatMatrix` or something like that.